sdk/testing/src/main/java/com/mesosphere/sdk/testing/SendOffer.java

```diff
@@ -139,35 +148,55 @@ public String getDescription() {
                 }
             }
         }
+
+        // In addition to the resources required by the tasks, add some resources for the Default Executor overhead.
+        if (!podToReuse.isPresent()) {
+            offerBuilder.addAllResources(DEFAULT_EXECUTOR_RESOURCES);
```

Comment:
How would this have to change if we wanted to test *both* default and custom executors?

Reply:
If custom executors are being exercised, the scheduler will effectively be getting slightly larger offers than it requires. It would ignore the additional cpu/mem/disk and launch the tasks anyway.